Tests/StreamChatTests/Models/Attachments/GiphyAttachmentPayload_Tests.swift

```diff
@@ -29,4 +29,19 @@ final class GiphyAttachmentPayload_Tests: XCTestCase {
         XCTAssertEqual(sut?.previewURL, giphyURL)
         XCTAssertEqual(sut?.actions, [])
     }
+    
+    func test_ItInitializesPayloadWithoutTitle() {
+        // WHEN
+        let title: String? = nil
+        sut = GiphyAttachmentPayload(
+            title: title,
```

Comment:
```suggestion
        sut = GiphyAttachmentPayload(
            title: nil,
```

Reply:
@polqf, this `let` is used in two places in the test 🤔

Reply:
Implemented `XCTAssertNil` 👍